website/docs/opengl/sources/1.3-final-result.md

```diff
@@ -0,0 +1,195 @@
+# Final source code of Chapter 1, Tutorial 3.
```

Comment:
Again this file should match up with the examples folder.

Reply:
I just removed the comments and unnecessary mipmap generation

Reply:
Ah I see the problem, it's just this tutorial that uses StbImageSharp. Apparently I let this slip through the cracks, the rest use SixLabors.ImageSharp which is the much better library in my opinion and would be what I want to recommend to our users.

In which case, this is fine for now and hopefully someone switches the library used here to be consistent in the future.

Reply:
if i have time i will try to make the switch in the future (because i will have to rewrite the original tutorial also), but thanks for the search